src/index.css

```diff
@@ -110,11 +110,11 @@ main {
   cursor: default;
 }
 
-.tachado{
+.NotLineThrough{
   text-decoration: none;
 }
 
-.setTachado{
+.setLineThrough{
```

Comment:
Apart from kebab-case naming, this is not the best name for the class. What if tomorrow I tell you to mark 'done' tasks with a red color or a yellow background instead of a line through? Change its name to something like 'done', 'completed', 'marked' or whatever you want, but not attached to its properties.